### PR TITLE
React Native Release 2.5.1

### PR DESCRIPTION
**Pull request by charmian01**
## Description
Bumped the versions
Updated TestApp to use latest SDKs
Updated the Changelog.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
 # App Center SDK for React Native Change Log
 
-## Version 2.5.1 (Under development)
+## Version 2.5.1
```
Comment (jaelim-ms) [inline CHANGELOG.md]: Should be 2.6.0.

File: TestApp/ios/Podfile.lock [hunk 2]
```diff
@@ -1,40 +1,40 @@
 PODS:
-  - appcenter (2.5.0):
+  - appcenter (2.5.1):
```
Comment (jaelim-ms) [inline TestApp/ios/Podfile.lock]: 2.6.0

File: TestApp/ios/Podfile.lock [hunk 3]
```diff
@@ -1,40 +1,40 @@
...
     - AppCenterReactNativeShared
     - React
-  - appcenter-analytics (2.5.0):
+  - appcenter-analytics (2.5.1):
     - AppCenter/Analytics
     - AppCenterReactNativeShared
     - React
-  - appcenter-auth (2.5.0):
+  - appcenter-auth (2.5.1):
```
Comment (jaelim-ms) [inline TestApp/ios/Podfile.lock]: 2.6.0

File: TestApp/ios/Podfile.lock [hunk 4]
```diff
@@ -1,40 +1,40 @@
 PODS:
-  - appcenter (2.5.0):
+  - appcenter (2.5.1):
     - AppCenterReactNativeShared
     - React
-  - appcenter-analytics (2.5.0):
+  - appcenter-analytics (2.5.1):
```
Comment (jaelim-ms) [inline TestApp/ios/Podfile.lock]: 2.6.0

File: TestApp/ios/Podfile.lock [hunk 5]
```diff
@@ -1,40 +1,40 @@
...
     - AppCenter/Analytics
     - AppCenterReactNativeShared
     - React
-  - appcenter-auth (2.5.0):
+  - appcenter-auth (2.5.1):
     - AppCenter/Auth
     - AppCenterReactNativeShared
     - React
-  - appcenter-crashes (2.5.0):
+  - appcenter-crashes (2.5.1):
```
Comment (jaelim-ms) [inline TestApp/ios/Podfile.lock]: 2.6.0

File: TestApp/ios/Podfile.lock [hunk 6]
```diff
@@ -1,40 +1,40 @@
...
     - AppCenter/Crashes
     - AppCenterReactNativeShared
     - React
-  - appcenter-data (2.5.0):
+  - appcenter-data (2.5.1):
```
Comment (jaelim-ms) [inline TestApp/ios/Podfile.lock]: 2.6.0

File: TestApp/ios/Podfile.lock [hunk 7]
```diff
@@ -1,40 +1,40 @@
...
     - AppCenter/Data
     - AppCenterReactNativeShared
     - React
-  - appcenter-push (2.5.0):
+  - appcenter-push (2.5.1):
```
Comment (jaelim-ms) [inline TestApp/ios/Podfile.lock]: 2.6.0

File: TestApp/ios/Podfile.lock [hunk 8]
```diff
@@ -1,40 +1,40 @@
...
     - AppCenter/Core
-  - AppCenter/Data (2.5.0):
+  - AppCenter/Data (2.5.1):
     - AppCenter/Core
-  - AppCenter/Push (2.5.0):
+  - AppCenter/Push (2.5.1):
     - AppCenter/Core
-  - AppCenterReactNativeShared (2.5.0):
-    - AppCenter/Core (= 2.5.0)
+  - AppCenterReactNativeShared (2.5.1):
```
Comment (jaelim-ms) [inline TestApp/ios/Podfile.lock]: 2.6.0

Comment (charmian01) [reply TestApp/ios/Podfile.lock]: "AppCenter/Core" version also should be 2.6.0, right?

File: CHANGELOG.md [hunk 9]
```diff
@@ -14,6 +18,10 @@
 
 * **[Fix]** Fix typescript compiler errors related to const enum usage.
 
+#### Android
+
+* **[Feature]** Add the `Crash.trackError` method to send handled errors (with optional properties and attachments).
```
Comment (dhei) [inline CHANGELOG.md]: This feature is only for native android sdk, not for RN sdk. We should remove it from changelog.

**Pull request merged**